qml/Ping1DControlPanel.qml

```diff
@@ -5,8 +5,6 @@ import QtQuick.Controls 2.2
 import QtQuick.Controls.Material 2.2
 import QtQuick.Layouts 1.3
 
-import DeviceManager 1.0
```

Comment:
This is necessary to get the connected device, otherwise the entire widget will not work.

Reply:
uuuuuff how did I miss this